Atomic/atomic.py

```diff
@@ -553,17 +553,17 @@ def cmd_env(self):
         os.environ['NAME'] = self.name or ""
         os.environ['IMAGE'] = self.image or ""
 
-        if hasattr(self.args, 'opt1') and self.args.opt1:
+        if self.args.get('opt1', False):
             os.environ['OPT1'] = self.args.opt1
```

Comment:
Hmm...not related to this change but this looks a bit dangerous - we're mutating the current process' environment, which means that conceptually we can only do one action at a time.  If we ever supported running multiple containers in one go, this would have to change.

It's normally better regardless to prepare the environment for the child process rather than mutating self's.  Anyways, not a blocker for this patch, just noting.